### PR TITLE
Fix FR formatting

### PR DESCRIPTION
**Pull request by skyman503**
This CL changes the formatting expression for address_overflow, making it go second after the building location

### CODE REVIEW

File: model/countries/FR/FR-parsing-rules.yaml [hunk 1]
```diff
@@ -16,6 +16,12 @@ regex_definitions:
   # Regular expression to match the prefixes that indicate a house number or name.
   kHouseNumberOrNameOptionalPrefixRe:
     regex_fragment: '(?:(?:no|nr|°|º|numéro)[-.\s]*)?'
+  
+  kCommaOrNewlineSeparator:
```
Comment (norgevz) [inline model/countries/FR/FR-parsing-rules.yaml:20]: Isn't this similar to `kCommaOrWhitespaceSeparator`? I'm concerned about introducing multiple different separators on multiple country files. Ideally, we use shared ones as most as possible.

File: model/countries/FR/FR-parsing-rules.yaml [hunk 2]
```diff
@@ -16,6 +16,12 @@ regex_definitions:
...
+    regex_fragment: '(,|, |\n)'
```
[the remark this answers is not in the document]
Comment (skyman503) [reply model/countries/FR/FR-parsing-rules.yaml:21]: Added a test case and I think it is still fine.

File: model/countries/FR/FR-parsing-rules.yaml [hunk 3]
```diff
@@ -36,14 +42,13 @@ capture_definitions:
     capture:
       output: street-address-alternative-1
       parts:
+      - capture_reference: ParseBuildingLocation
+      - separator: {regex_reference: kCommaOrNewlineSeparator}
       - capture:
           output: address-overflow
-          parts: [ {regex_fragment: '(?:[^,\r\n]+)'} ]
+          prefix: {regex_reference: kSpaceOptionalPrefixRe}
```
Comment (norgevz) [inline model/countries/FR/FR-parsing-rules.yaml:49]: How about just inlining `{regex_fragment: '\s*' }`?

File: model/countries/FR/FR-parsing-rules.yaml (hunk 1 above)
Comment (skyman503) [reply model/countries/FR/FR-parsing-rules.yaml:20]: French street names can have spaces in them, the kCommaOrWhitespaceSeparator ends up matching the first space and thus leading to the wrong parsing of the Place Charles de Gaulle

File: model/countries/FR/FR-parsing-rules.yaml (hunk 3 above)
Comment (skyman503) [reply model/countries/FR/FR-parsing-rules.yaml]: Sg.

File: model/countries/FR/FR-parsing-rules.yaml (hunk 1 above)
Comment (norgevz) [reply model/countries/FR/FR-parsing-rules.yaml:20]: I see. Then probably you want something like `(?:, |\n|\r|,)+`?

File: model/countries/FR/FR-parsing-rules.yaml [hunk 4]
```diff
@@ -84,64 +86,73 @@ test_parsing_definitions:
...
     street: "rue du Gue Jacquet"
     building: "58"
     address-overflow: "App 1"
 - id: "Test 10"
   type: street-address-alternative-1
-  input: "Appt 1, 58 rue du Gue Jacquet"
+  input: "58 rue du Gue Jacquet, Appt 1"
   output:
-    street-address-alternative-1: "Appt 1, 58 rue du Gue Jacquet"
+    street-address-alternative-1: "58 rue du Gue Jacquet, Appt 1"
```
Comment (norgevz) [inline model/countries/FR/FR-parsing-rules.yaml:136]: Can you drop the space after the comma? I would like to validate that case as well.

**Review by norgevz — APPROVED** [review]
LGTM % a couple of updates.

File: model/countries/FR/FR-parsing-rules.yaml (hunk 4 above)
Comment (skyman503) [reply model/countries/FR/FR-parsing-rules.yaml]: Done.

**Pull request merged**